spec/integration/rspec/match_matcher_spec.rb

```diff
@@ -341,6 +341,48 @@
     end
   end
 
+  # HINT: `a_hash_including` is an alias of `include` in the rspec-expectations gem.
+  #       `hash_including` is an argument matcher in the rspec-mocks gem.
+  context "when the expected value is `hash-including-<something>`, not `a-hash-including-<something>`" do
```

Comment:
This is a simple copy of `hash_including-<something>` spec.
https://github.com/mcmire/super_diff/blob/c98b483c0c8d7787f1eb0d57de913ce12849b370/spec/integration/rspec/match_matcher_spec.rb#L6-L42